src/components/buttons/button-color.jsx

```diff
@@ -114,8 +114,8 @@ class ButtonColor extends React.Component {
 					onClick={toggleDropdown}
 					role="combobox"
 					tabIndex={tabIndex}>
-					<span className={activeColorClass}>
-						<ButtonIcon symbol="text-editor" />
+					<span>
+						<ButtonIcon symbol="color-picker" />
```

Comment:
Thanks @sandrodw3!

Are we sure we want to use the bucket icon for text color? It is often used to indicate background/fill color:

![fill](https://user-images.githubusercontent.com/7074/56588881-859b3100-65e4-11e9-857e-b129223c63b7.png)

with something else used for text:

![text](https://user-images.githubusercontent.com/7074/56588897-8b911200-65e4-11e9-8d43-be8323602464.png)